File: codes/julia/008_operator_ternary.jl
println("\n# Ternary Operator in Julia")

CORRECT_ANSWER::Any = "foo"
println("correct answer: $(CORRECT_ANSWER)")

my_answer::Any = "bar"
println("my answer: $(my_answer)")
println("is my answer correct: $(((my_answer === CORRECT_ANSWER) ? "correct!" : "wrong!"))")
# is my answer correct: wrong!

my_answer::Any = "foo"
println("my answer: $(my_answer)")
println("is my answer correct: $(((my_answer === CORRECT_ANSWER) ? "correct!" : "wrong!"))")
# is my answer correct: correct!

my_answer::Any = "bar"
println("my answer: $(my_answer)")
println("is my answer correct: $(ifelse((my_answer === CORRECT_ANSWER), "correct!", "wrong!"))")
# is my answer correct: wrong!

my_answer::Any = "foo"
println("my answer: $(my_answer)")
println("is my answer correct: $(ifelse((my_answer === CORRECT_ANSWER), "correct!", "wrong!"))")
# is my answer correct: correct!

File: codes/javascript/src/008_operator_ternary.js
console.log("\n// Ternary Operator in JavaScript");

const CORRECT_ANSWER = "foo";
console.log(`correct answer: "${CORRECT_ANSWER}"`);

let myAnswer;

myAnswer = "bar";
console.log(`my answer: "${myAnswer}"`);
console.log(`is my answer correct: ${((myAnswer === CORRECT_ANSWER) ? "correct!" : "wrong!")}`);
// is my answer correct: wrong!

myAnswer = "foo";
console.log(`my answer: "${myAnswer}"`);
console.log(`is my answer correct: ${((myAnswer === CORRECT_ANSWER) ? "correct!" : "wrong!")}`);
// is my answer correct: correct!
